lace_fake/snapshot_admin.py

```diff
@@ -24,14 +24,21 @@ def __init__(self,snap_json,scales_tau=None,kF_Mpc=None,post_dir=None):
             print('use',post_dir,'as post-processing directory')
             self.data['post_dir']=post_dir
             self.data['raw_dir']=None
-            self.data['skewers_dir']=post_dir+'/skewers/'
         else:
             if 'post_dir' in self.data:
                 print('no post_dir provided, will use',self.data['post_dir'])
-                self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
+                # self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
             else:
                 raise ValueError('must provide post_dir when using old files')
 
+        self.data['axis'] = axis
```

Comment:
I think we can delete this line, since we are assigned it again in the if/else code block.

Reply:
Yes, I was doing something different before, good catch!